test/metabase/api/database_test.clj

```diff
@@ -1329,7 +1331,7 @@
         (testing "Should throw 404 if the schema/Collection doesn't exist"
           (is (= "Not found."
                  (mt/user-http-request :lucky :get 404
-                                       (format "database/%d/schema/Coin Collection" lib.schema.id/saved-questions-virtual-database-id)))))))
+                                       (format "database/%d/schema/%s" lib.schema.id/saved-questions-virtual-database-id "Coin%20Collection")))))))
```

Comment:
Can't we just have the mock client URL-escape spaces?

Reply:
yea you're right 00fc2ce55bb3f10c6643ee0e02a2992b05eff532